golang/x/relay/client/rest/query.go

```diff
@@ -263,9 +263,9 @@ func getRequestHandler(cliCtx context.CLIContext, storeName string) http.Handler
 	return func(w http.ResponseWriter, r *http.Request) {
 		vars := mux.Vars(r)
 
-		id, err := strconv.ParseUint(vars["id"], 10, 64)
-		if err != nil {
-			rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+		id, sdkErr := types.RequestIDFromHex(vars["id"])
```

Comment:
should this also use `requestIDFromString` ? if so, should we move it to the `types` folder?

Reply:
Oh, I forgot I wrote that.  Yeah, let's move it to the types folder

Reply:
Pushed up a change, want to take a look?